docs/geedocs/5.2.3/answer/6051700.html

```diff
@@ -126,6 +126,33 @@ <h3>Symbolic links</h3>
   <li><strong>soft-link</strong>: the disconnected database folder and published_dbs folder are on different logical volumes.</li>
   <li>If the disconnected database folder is in <code>/tmp</code>, the directory files will be copied. </li>
 </ul>
+<h3>Rewrite KML Layer URLs</h3>
+<p>If the disconnected database contains KML Layers, the KML Layer's URLs may require adjustments in order to resolve the KML content in a disconnected environment.   (See <a href="../answer/2760457.html">Add KML Layers</a>). Use the <code>gerewritedbroot</code> command to update the disconnected database KML Links with new hostname, ports, paths or filenames. </p>
+<ol>
+  <li>Publish KML Content
+    <p>Copy and publish and modify the KML files on the disconnected GEE Server machine's network as needed. Validate that the referenced KML files are accessible on the disconnected network via HTTP or HTTPS using a web browser or command line tools. Note the differences in hostnames, paths and ports between their origninal locations and their locations on the disconnected network environment. </p>
+    <p>KML files can be served from the <code>/opt/google/gehttpd/htdocs/</code> folder or a subfolder, or on a separate web server. </p>
+  </li>
+
+  <li>Rewrite the KML Layer Links in the database on the Server. 
```

Comment:
I'm not sure this makes sense as a step because the steps after it describe how to do it.